using-kubecost/navigating-the-kubecost-ui/teams.md

```diff
@@ -23,79 +22,236 @@ persistentVolume:
+
+An example values snippet is shown below for the creation of a team in Helm.
 
 ```yaml
-saml:
-  enabled: true
-  rbac:
-    enabled: false
+ teams:
+   # teamsConfigMapName: teams-configmap-name
+   teamsConfig:
+     - id: ''
+       name: helm-team
+       roles:
+       - id: ''
+         name: helm-role
+         description: helm configured role
+         pages:
+           showOverview: true
+           showAllocation: true
+           showAsset: true
+           showCloudCost: true
+           showClusters: true
+           showExternalCosts: true
+           showNetwork: true
+           showCollections: true
+           showReports: true
+           showInsights: true
+           showActions: true
+           showAlerts: true
+           showBudgets: true
+           showAnomalies: true
+           showEfficiency: true
+           showSettings: true
+         permissions: admin
+         routes: []
+         allocationFilters:
+         - key: cluster
+           operator: ":"
+           value: cluster-one
+         assetFilters: []
+         cloudCostFilters: [] 
+       claims:
+         NameID: email@domain.com
 ```
 
-Now, access the Teams page, and create your initial admin team. See the [Adding a team](teams.md#adding-a-team) section below for specific instructions for using the UI. Once your admin team has been created, reenable SAML RBAC authentication:
+
+
+Configure at least one team under `teamsConfig` with an associated role under the team's `roles`.
+
+Alternatively, a ConfigMap can be created manually, with `teamsConfigMapName` set to this ConfigMap's name.
 
 ```yaml
-saml:
-  enabled: true
-  rbac:
-    enabled: true
+apiVersion: v1
+data:
+  rbac-teams-configs.json: '[json teams data]'
+kind: ConfigMap
+metadata:
+  name: <your-configmap-name>
+  namespace: <your-kc-namespace>
 ```
 
-The members of your team are now able to add new members, or create new teams.
+Note that setting a manually-configured ConfigMap name will override `teamsConfig`.
+
+For more information about teams and roles, see the [Adding a team](teams.md#adding-a-team) section below.
+
+## Claims
+
+Claims are part of the Identity Provider (IdP) response that provide information about the authenticated user. For SAML, these are relayed as part of the SAML response. For OIDC, these are returned as part of the access or ID tokens.
+
+Kubecost Teams uses these claims to map users to teams. As long as a group or attribute defined in your IdP is returned as part of the authorization process in the SAML response/OIDC token(s), it can be used in teams.
+
+For example, in OIDC, an ID token may contain a certain number of claims.
+
+```json
+{
+  "sub": "11111111",
+  "name": "John Example",
+  "email": "john.example@company.com",
+  "iss": "https:/authserver.com",
+  "groups": {
+    "kubecost_admins",
+  }

```

Comment:
```suggestion
  "groups": {
    "kubecost_admins",
  },
```

Also, double check this please. `groups` looks like the data type is array/string and not object as implied here by the braces. I know it may not be part of everyones' ID token, but we should still represent it correctly.

Reply:
Fixed!